README.md

```diff
@@ -3,43 +3,46 @@
 [![Build Status](https://travis-ci.org/firebase/firepad.svg?branch=master)](https://travis-ci.org/firebase/firepad)
 [![Version](https://badge.fury.io/gh/firebase%2Ffirepad.svg)](http://badge.fury.io/gh/firebase%2Ffirepad)
 
-[Firepad](http://www.firepad.io/) is an open-source, collaborative code and text editor. It was
-designed to be embedded inside larger applications. Since it uses
-[Firebase](https://www.firebase.com/?utm_source=firepad) as a backend, it requires no server-side
-code and can be added to any web app simply by including a couple JavaScript files.
+[Firepad](http://www.firepad.io/) is an open-source, collaborative code and text editor. It is
+designed to be embedded inside larger web applications. 
 
-Visit [firepad.io](http://www.firepad.io/) for a demo as well as instructions on how to embed it
-into your application!
+## Live Demo
 
-Join our [Firepad Google Group](https://groups.google.com/forum/#!forum/firepad-io) to ask
-questions, request features, or share your Firepad apps with the community.
+Visit [firepad.io](http://demo.firepad.io/) to see a live demo of Firepad in rich text mode, or the
+[examples page](http://www.firepad.io/examples/) to see it setup for collaborative code editing.
 
+[![a screenshot of demo.firepad.io including a picture of two cats and a discussion about fonts](screenshot.png)](http://demo.firepad.io/)
 
-## Contributing
+## Setup
+Firepad uses [Firebase](https://www.firebase.com/?utm_source=firepad) as a backend, so it requires no server-side
+code. It can be added to any web app by including a few JavaScript files
 
-If you'd like to contribute to Firepad, you'll need to run the following commands to get your
-environment set up:
+	<!-- Firebase -->
+	<script src="https://cdn.firebase.com/js/client/2.0.2/firebase.js"></script>
 
-```bash
-$ git clone https://github.com/firebase/firepad.git
-$ cd firepad                # go to the geofire directory
-$ npm install -g grunt-cli  # globally install grunt task runner
-$ npm install -g bower      # globally install Bower package manager
-$ npm install               # install local npm build / test dependencies
-$ bower install             # install local JavaScript dependencies
-$ grunt watch               # watch for source file changes
-```
+	<!-- ACE and its JavaScript mode and theme files -->
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/ace/1.1.3/ace.js"></script>
```

Comment:
The CodeMirror version is better tested and more often used (plus it can support rich-text)...  so I'd be inclined to include it as the example instead of ACE...